.buildkite/bk.integration.pipeline.yml

```diff
@@ -53,15 +54,42 @@ steps:
           - fleet-privileged
           - upgrade
 
-  - group: "Stateful(Sudo): Ubuntu"
+      - label: "Win2022:non-sudo:{{matrix}}"
+        command: |
+          buildkite-agent artifact download build/distributions/** . --step 'package-it'
+          .buildkite/scripts/integration-tests.ps1 {{matrix}} false
+        artifact_paths:
+          - build/**
+        agents:
+          provider: "gcp"
+          machineType: "n1-standard-8"
+          image: "family/platform-ingest-elastic-agent-windows-2022"
+        matrix:
+          - default
+
+  - group: "Stateful:Ubuntu"
     key: integration-tests-ubuntu
     depends_on:
       - integration-ess
+      - package-it
     steps:
-      - label: "x86_64:Group: {{matrix}}"
+      - label: "x86_64:non-sudo: {{matrix}}"
```

Comment:
The default jobs list view in BK doesn't include the group name, so maybe it'd be good to prefix these step labels with `Ubuntu`? It'd match how the Windows step labels start. The groups only get shown in BK's canvas and waterfall views.

Default list view:
<img width="297" alt="image" src="https://github.com/user-attachments/assets/1fa31d07-ad78-42f2-91dc-443a933f26b7">

Canvas view:
<img width="466" alt="image" src="https://github.com/user-attachments/assets/88d15f12-219f-4038-b0d9-9872ecbfba6e">

Reply:
+1 to drop the groups and move the group name into the step as a prefix; this makes the default UI experience in CI much easier to understand. Until BK comes and makes changes to how groups/steps are shown.

Reply:
Raised a follow-up issue: https://github.com/elastic/ingest-dev/issues/4518